### PR TITLE
Release 39 merge

### PR DESCRIPTION
**Pull request by rprouse**
Merges the changes from the 3.9 release and increments the version.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+and `ApartmentState` not being properly applied to tests in all cases.
+
+#### Issues Resolved
+
+ * 893 Inconsistent Tuple behavior.
+ * 1239 NUnit3 sometimes hangs if SetUpFixtures are run in parallel
+ * 1346 NullReferenceException when [TestFixtureSource] refers to data in a generic class.
+ * 1473 Allow Is.Ordered to Compare Null Values
+ * 1899 Constraint Throws.Exception does not catch exception with async lambdas
+ * 1905 SetupFixture without namespace will make assembly-level Parallelable attribute useless
```
Comment (jnm2) [inline CHANGES.md]: Parallelable :-/ do we care about issue titles?

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+ * 2441 Allows to override load-time/execution-time interfaces in built-in tests attributes
+ * 2446 CI failure in mono Warning tests
+ * 2448 Inherited Test SetUp, TearDown, etc. are not executed in .NET Core if they are not public
+ * 2451 Compile RegEx to improve performance
+ * 2454 SetUpFixture not respecting NonParallelizable tag on TestFixtures.
+ * 2459 [Parallelizable(ParallelScope.Children)] Unable to finish tests
+ * 2465 Possible wrong properties are returned by reflection in ReflectionExtensions.cs
+ * 2467 Test execution hangs when using [SetUpFixture] with NUnit 3.8.x
+ * 2469 Allow RangeAttribute to be specified multiple times for the same argument
+ * 2471 Parametrized testcases not running in parallel
```
Comment (jnm2) [inline CHANGES.md]: Parametrized :-/

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,64 @@
...
+ * 2497 Use ConstraintUtils.RequireActual through out the codebase
+ * 2504 Support changing test display name on TestFixtureData
+ * 2508 Correct divergence from shadowed Is / Has members.
+ * 2516 When test writes something to the stdErr there is no guaranteed way to link a test-output event to a target test using ITestEventListener
+ * 2525 Remove unwanted space from comment
+ * 2526 SerializationException in low trust floating point equality test
+ * 2533 Matches<T>(Predicate<T>) throws ArgumentException or Fails when actual is null
+ * 2534 SetUpFixture causes NUnit to lock with Apartment( STA )
+ * 2551 CollectionItemsEqualConstraint is missing Using(Func<T, T, bool>)
+ * 2554 Made TestFixtureData.SetName internal for 3.9
```
Comment (jnm2) [inline CHANGES.md]: Reversal of #2504

**Comment by rprouse** [comment]
All of the issue titles are automatically generated. I went through and cleaned a few up in GitHub then regenerated, but missed a few 😄

**Pull request merged**